ledger-tool/src/args.rs

```diff
@@ -53,6 +53,13 @@ pub fn accounts_db_args<'a, 'b>() -> Box<[Arg<'a, 'b>]> {
             .validator(is_pow2)
             .takes_value(true)
             .help("Number of bins to divide the accounts index into"),
+        Arg::with_name("accounts_index_initial_accounts")
+            .long("accounts-index-initial-accounts")
+            .value_name("NUMBER")
+            .validator(is_parsable::<usize>)
+            .takes_value(true)
+            .help("Pre-allocate the accounts index, assuming this many number of accounts")
```

Comment:
Apologies, I think my suggestion's wording was a little strange. I think *either*:

"this many accounts"

OR

"this number of accounts"